modules/ibi-generate-seed-image.adoc

```diff
@@ -0,0 +1,175 @@
+* Generating new certificates for the seed cluster
+* Restoring and updates the `SeedGenerator` CR on the seed cluster
+
+[NOTE]
+====
+The generated seed image does not include any site-specific data.
+
+To avoid including any {rh-rhacm-first} resources in your seed image, you need to disable all optional {rh-rhacm} add-ons for seed image generation.
+====
+
+.Prerequisites
+
+* You have deployed a managed a {sno} cluster with a DU profile.
+* Install the {lcao} in the hub cluster.
+* Log in as a user with `cluster-admin` privileges.
+* Review the required versions of software components in the section _Software prerequisites for the image-based installation_.
+* You created an an extra partition in the seed cluster. See _Configuring a shared container directory between `ostree` stateroots_ for further information.
+
+.Procedure
+
+. Detach the cluster to delete any cluster-specific resources from the seed cluster that must not be in the seed image:
+
+.. Manually detach the seed cluster by running the following command:
++
+[source,terminal]
+----
+$ oc delete managedcluster sno-worker-example
+----
+
+... Wait until the `ManagedCluster` CR is removed. Once the CR is removed, create the proper `SeedGenerator` CR. The {lcao} cleans up the {rh-rhacm} artifacts.
+
+. Create the `Secret`.
+
+.. Create the authentication file by running the following command:
++
+--
+.Authentication file
+[source,terminal]
+----
+$ MY_USER=myuserid
+$ AUTHFILE=/tmp/my-auth.json
+$ podman login --authfile ${AUTHFILE} -u ${MY_USER} quay.io/${MY_USER}
+----
+
+[source,terminal]
+----
+$ base64 -w 0 ${AUTHFILE} ; echo
+----
+--
+
+.. Copy the output into the `seedAuth` field in the `Secret` YAML file named `seedgen` in the `openshift-lifecycle-agent` namespace.
++
+--
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: seedgen <1>
+  namespace: openshift-lifecycle-agent
+type: Opaque
+data:
+  seedAuth: <encoded_AUTHFILE> <2>
+----
+<1> The `Secret` resource must have the `name: seedgen` and `namespace: openshift-lifecycle-agent` fields.
+<2> Specifies a base64-encoded authfile for write-access to the registry for pushing the generated seed images.
+--
+
+.. Apply the `Secret`.
++
+[source,terminal]
+----
+$ oc apply -f secretseedgenerator.yaml
+----
+
+. Create the `SeedGenerator` CR:
++
+--
+[source,yaml]
+----
+apiVersion: lca.openshift.io/v1
+kind: SeedGenerator
+metadata:
+  name: seedimage <1>
+spec:
+  seedImage: <seed_container_image> <2>
+----
+<1> The `SeedGenerator` CR must be named `seedimage`.
+<2> Specify the container image URL, for example, `quay.io/example/seed-container-image:<tag>`. It is recommended to use the `<seed_cluster_name>:<ocp_version>` format.
+--
+
+. Generate the seed image by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f seedgenerator.yaml
+----
+
++
+[IMPORTANT]
+====
+The cluster reboots and loses API capabilities while the {lcao} generates the seed image.
```

Comment:
Maybe clarify what cluster reboot means here? FWIU the node does not reboot during seed image generation

Reply:
````
During the seed generation process, the cluster temporarily loses API capabilities because {lcao} stops the `kubelet` service and shuts down all running Operators and pods. When the process completes, {lcao} restarts the `kubelet` service, which triggers the recovery of the Operators. 
````
How about this?

Reply:
That new phrase is accurate :)
Maybe we can talk about `stops/starts openshift` instead of kubelet? WDYT?

Reply:
Updated with stop/start OpenShift terminology.